README.md

```diff
@@ -1,3 +1,6 @@
 # workspace
 
 Themes, utilities and settings to optimize our development workflow
+
+#Setup
```

Comment:
Invalid markdown, needs space between hash and title. Also I would recommend ## as this is a sub-heading to # workspace